_content/tour/ind/algorithms-numbers.article

```diff
@@ -1,31 +1,31 @@
-Number Operations
-This section provides examples that perform number operations.
+Operasi Bilangan
+Bagian ini memberikan contoh mengenai operasi bilangan.
 
-* Palindrome
+* Palindrom
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Tonton Video]]
+- Perlu bantuan finansial, Isi [[https://www.ardanlabs.com/scholarship/][Formulir Beasiswa]]
 
-The sample program implements a check to see if an integer is a
-palindrome or not.
+Contoh program berikut ini mengimplementasikan pengecekan
```

Comment:
Instead of `mengimplementasikan pengecekan` how about `melakukan pengecekan` ?
I think the second one gives better understanding than the first one